### DISCUSSION TITLE
Webhooks verification challenge and authorization

### DISCUSSION THREAD

**Discussion by h0lyluck** [fattureincloud/api#422] [Ideas]
Hi, I was trying to activate the webhook but failed the verification challenge because i was first validating the authorization [as per documentation](https://developers.fattureincloud.it/docs/webhooks/notifications#policeman-validate-the-requests) but failed because the verification request was made without the authorization header.
Maybe adding the authorization header to the verification request will be helpful to others :)

**Comment by valmoz** [comment]
Hi @tcode92 
thank you for this suggestion, it surely would be an improvement to our Webhooks functionality, but we don't think it is a priority since this request is only sent after a subscription creation and shouldn't trigger any complex computation.

We'll add it to our backlog, but I can't give you any release date for now.

Thanks
Mauro

**Reply by h0lyluck** [reply]
Hi @valmoz thanks for the replay.
I totally agree about the computation part, and i don't expect this to be present at all, just adding it in the documentation should be totally fine.
